test/integration/connectors/expected_results/s3-specialchar/file_data/869bf15f-e840-51dc-a818-8d0b817817c9.json

```diff
@@ -0,0 +1,37 @@
+{
+    "identifier": "869bf15f-e840-51dc-a818-8d0b817817c9",
+    "connector_type": "s3",
+    "source_identifiers": {
+      "filename": "Why_is_the_sky_blue?.txt",
+      "fullpath": "utic-dev-tech-fixtures/special-characters/Why_is_the_sky_blue?.txt",
```

Comment:
I created a new text file and reproduced the bug with this file. Then, I uploaded it here:
https://us-east-2.console.aws.amazon.com/s3/object/utic-dev-tech-fixtures?region=us-east-2&bucketType=general&prefix=special-characters/Why_is_the_sky_blue%3F.txt